File: Math/excelColumnTitle.js

function excelColumnTitle(A) {
    var startCharCode = 'A'.charCodeAt(0) - 1;
    result = '';
    
    while (A>0) {
        var value = A % 26
        if (value === 0) {
            result += 'Z'
            A = Math.floor(A/26) - 1
        } else {
            result += String.fromCharCode(value + startCharCode)
            A = Math.floor(A/26)
        }
    }
    return result.split('').reverse().join('')

}

var a = 676
console.log(excelColumnTitle(a))

File: Math/excelColumnTitle.py


def excelColumnTitle(A):
    startCharCode = ord('A') - 1
    result = ''
    while(A > 0):
        value = (A % 26)
        print(value)
        if value == 0:
            result += 'Z'
            A = (A // 26) - 1
        else:
            result += chr(value + startCharCode)
            A = A // 26
    return result[::-1]

a = 676+26
print(excelColumnTitle(a))